evm-template/runtime/src/configs/xcm_config.rs

```diff
@@ -109,6 +109,28 @@ pub type XcmOriginToTransactDispatchOrigin = (
     XcmPassthrough<RuntimeOrigin>,
 );
 
+parameter_types! {
+    /// Xcm fees will go to the treasury account
+    pub XcmFeesAccount: AccountId = Treasury::account_id();
+}
+
+/// This is the struct that will handle the revenue from xcm fees
+/// We do not burn anything because we want to mimic exactly what
+/// the sovereign account has
+pub type XcmFeesToAccount = xcm_primitives::XcmFeesToAccount<
```

Comment:
we have a specific file for storing `types`. I think it's better to move this to there for consistency

Reply:
Yeah we can discuss that in the meeting. I have no strong preference for that. If we want to put types near where they are used, I'm ok with the change. I can handle that PR in the future